src/Backups/BackupFileInfo.cpp

```diff
@@ -49,44 +49,19 @@ namespace
     struct ChecksumsForNewEntry
     {
         UInt128 full_checksum;
-        UInt128 prefix_checksum;
+        std::optional<UInt128> prefix_checksum;
```

Comment:
Do we need to differentiate between `0` and `std::nullopt`? It seems that in current code `prefix_checksum` cannot be 0.

Reply:
Changed the type to ordinary UInt128.